bootstrapvz/common/task_groups.py

```diff
@@ -33,7 +33,16 @@ def get_standard_groups(manifest):
 	group.extend(get_network_group(manifest))
 	group.extend(get_apt_group(manifest))
 	group.extend(security_group)
+
+	from bootstrapvz.common.releases import jessie
+	if manifest.release > jessie:
+		if not locale.SetLocalTimeLink in locale_group:
+			locale_group.extend(locale.SetLocalTimeLink)
```

Comment:
`extend()` only takes another list as an argument. You want to use `append`.
Also, `standard_groups()` is really not meant for adding individual tasks. The tasks you got are a part of the `locale_group`, which right now is a simple list. Convert that group to a function like `get_network_group()`.